web/core/components/stickies/delete-modal.tsx

```
@@ -0,0 +1,44 @@
+
+  const formSubmit = async () => {
+    await handleSubmit();
+    setLoader(true);
+    try {
+    } catch (error) {
+      setToast({
+        type: TOAST_TYPE.ERROR,
+        title: "Warning!",
+        message: "Something went wrong please try again later.",
+      });
+    }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)